src/dvclive/lightning.py

```diff
@@ -50,36 +50,19 @@ def _should_call_next_step():
 
 
 class DVCLiveLogger(Logger):
-    def __init__(  # noqa: PLR0913
+    def __init__(
         self,
         run_name: Optional[str] = "dvclive_run",
         prefix="",
         log_model: Union[str, bool] = False,
         experiment=None,
-        dir: Optional[str] = None,  # noqa: A002
-        resume: bool = False,
-        report: Optional[str] = None,
-        save_dvc_exp: bool = False,
-        dvcyaml: bool = True,
-        cache_images: bool = False,
-        exp_name: Optional[str] = None,
+        **kwargs,
     ):
         super().__init__()
         self._prefix = prefix
-        self._live_init: Dict[str, Any] = {
-            "resume": resume,
-            "report": report,
-            "save_dvc_exp": save_dvc_exp,
-            "dvcyaml": dvcyaml,
-            "cache_images": cache_images,
-        }
-        if dir is not None:
-            self._live_init["dir"] = dir
+        self._live_init: Dict[str, Any] = kwargs
```

Comment:
double check:

- default values in kwargs, e.g. will we still consider `dvcyaml` as True if it's not specified in kwargs
- might worth creating copy of kwargs, so that we 100% that's it's frozen for the external world

Reply:
Added tests for both of those cases. With the test, I didn't think it's necessary to make a copy of kwargs, and I don't see it happening in other loggers like https://github.com/Lightning-AI/lightning/blob/master/src/lightning/pytorch/loggers/comet.py, but not that strong an opinion if you really feel it's necessary.

Reply:
yeah, I think we overdid this :) kwargs creates a new dict object when it's being passed I think we don't need a test (no need to test Python I guess :) ). I was asking primarily to check that tbh

same with default values, if it's Python behavior - no need to test this